lisa/tests/base.py

```diff
@@ -835,19 +837,74 @@ def get_trace(self, **kwargs):
         """
         return Trace(self.trace_path, self.plat_info, **kwargs)
 
-class RTATestBundle(FtraceTestBundle):
+class DmesgTestBundle(TestBundle):
     """
-    Abstract Base Class for :class:`lisa.wlgen.rta.RTA`-powered TestBundles
-
-    .. seealso: :class:`lisa.tests.base.FtraceTestBundleMeta` for default
-        ``ftrace_conf`` content.
+    Abstract Base Class for TestBundles based on dmesg output.
     """
-
```

Comment:
Spurious change? I think this newline made sense.

Reply:
It's actually not a change per say as I'm adding new code, not modifying it (don't be confused by diff, there are gazillions of equivalent patches, and diff just picks one that is usually reasonable, but not always).
I've added the newline in the new class.